code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -248,6 +248,37 @@
 	new /obj/item/weapon/grenade/chem_grenade/cleaner(src)
 	new /obj/item/weapon/grenade/chem_grenade/cleaner(src)
 
+/obj/item/weapon/storage/belt/lazarus
+	name = "trainer's belt"
+	desc = "For the mining master, holds your lazarus capsules."
+	icon_state = "lazarusbelt"
+	item_state = "lazbelt"
+	w_class = 4
+	max_w_class = 1
+	max_combined_w_class = 6
+	storage_slots = 6
+	can_hold = list("/obj/item/device/mobcapsule")

```

Comment:
Well, he could have it just count the number of capsules in the belt's contents and use THAT number instead of just the number of contents.

something like:
```
var/amount = 0
for(var/obj/O in contents)
     if(istype(O, /obj/item/device/mobcapsule))
          amount++
icon_state = "[initial(icon_state)]_[amount]"
```